optuna/cli.py

```diff
@@ -1,17 +1,12 @@
 """Optuna CLI module.
-
-This module is implemented using cliff. It follows
-[the demoapp](https://docs.openstack.org/cliff/latest/user/demoapp.html).
-
-If you want to add a new command, you also need to update `entry_points` in `setup.py`.
-c.f. https://docs.openstack.org/cliff/latest/user/demoapp.html#setup-py
+If you want to add a new command, you also need to update the constant `_COMMANDS`
 """
-
 from argparse import ArgumentParser  # NOQA
 from argparse import Namespace  # NOQA
```

Comment:
I guess this `# NOQA` is no `longer` needed.
```suggestion
from argparse import ArgumentParser
from argparse import Namespace
```